### ISSUE TITLE
UAAG requirements relevant for WCAG 2.2?

### ISSUE THREAD

**Issue by JAWS-test** [w3c/wcag#1013]
There are some requirements for browsers in the [UAAG ](https://www.w3.org/TR/UAAG20/)regarding HTML standard elements. If an author does not use standard elements, the browser cannot be held responsible for errors. The corresponding criteria should be included in WCAG 2.2 or 3.0.

An example:
* UAAG requires in 1.1.6 and 1.1.7 the extensive configuration of subtitles in videos.
* In the WCAG these are explicitly excluded from configuration (e.g. [SC 1.4.4](https://www.w3.org/TR/WCAG21/#resize-text)).

If the `<video>` element is not used with the `<track>` element, the browser cannot customize subtitles. Then it would be up to the author, right?

**Comment by patrickhlauke** [comment]
The point is it's irrelevant whether the author uses something that is provided directly by the user agent, or they make their own custom solution. For UAAG it's logical to say "the browser can't be held responsible for things if the authors doesn't use this particular standard element". For WCAG, which only cares about the end result for the user, this sort of requirement is unnecessary, as it's not based purely on providing browser-native solutions.

**Comment by alastc** [comment]
Agree with Patrick, and this sort of thing is covered by '[accessibility supported](https://www.w3.org/TR/WCAG21/#dfn-accessibility-supported)'. 

I'm assuming this was essentially a question and it is now answered, if not please re-open.

**Comment by JAWS-test** [comment]
I think my question was not properly understood. Because there are many things that the WCAG is not asking for at the moment that the UAAG is explicitly asking for. As long as standard elements are used, it's okay to hold the user agents responsible for accessibility. But if no standard elements are used, the authors should be responsible. This would only have to do with "accessibility supported" if these requirements were defined in the WCAG. Then the authors could choose the default element (if the user agent fulfills the requirement) or fulfill the requirement themselves. However, as long as the WCAG does not implement the requirements of the UAAG for custom elements, there will be a gap.

**Comment by alastc** [comment]
> if no standard elements are used, the authors should be responsible.

A lot of those cases are covered. For example, if you use a `<span>` for a `<button>`, you are responsible for the keyboard accessibility, the name/role value etc.

There may be things like captions that have more customisation requirements in UAAG than WCAG.

However, we'd have to deal with that on a case by case basis, and in that case there are so many instances of captions not being included, the primary thing is to include captions...
